extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/kernel/controller/ControllerStagePhase.java

```diff
@@ -35,7 +35,7 @@
     @Override
     public boolean canTransitionFrom(final ControllerStagePhase priorPhase)
     {
-      return false;
+      return true;
```

Comment:
`NEW` should be the initial state and therefore initialized when we are creating the `ControllerStagePhase`. We shouldn't allow any transition to `NEW` I think.

Reply:
It should not. Seems like leftover code.